src/integer.h

```diff
@@ -33,7 +33,7 @@ EXPORT_INLINE Int IS_LARGEINT(Obj obj)
 **  'IS_INT' returns 1 if 'obj' is either a large or an immediate integer
```

Comment:
So the return value of the function now is `BOOL`, should this comment be updated to 
```suggestion
**  'IS_INT' returns TRUE if 'obj' is either a large or an immediate integer
```

Reply:
There are a bunch of places where this is the case. I don't point them all out now because I first would like to hear your opinion on this.

Reply:
I am reluctant to touch them because to preserve the line width limit one has to rewrap most of these comments, and it didn't seem worth the pain. But I wouldn't object if somebody else wanted to do this in another PR once this one is merged  :-)